packages/back-end/src/services/python.ts

```diff
@@ -19,6 +19,9 @@ type PythonServerResponse<T> = {
 // The stats engine usually finishes within 1 second
 // We use an overly conservative timeout to account for high load
 const STATS_ENGINE_TIMEOUT_MS = 60_000;
+const MIN_POOL_SIZE = process.env.GB_STATS_ENGINE_POOL_SIZE_MIN
+  ? Number(process.env.GB_STATS_ENGINE_POOL_SIZE_MIN)
+  : 1;
```

Comment:
Will need validation to make sure:
1. It is a valid number (not `NaN`)
2. It is not negative
3. It is less than or equal to the max pool size